### ISSUE TITLE
Compilation error the provider "terraform-aws-eks"

### ISSUE THREAD

**Issue by hashir91** [johandry/terranova-examples#5]
I was trying out **"terraform-aws-eks"** provider with terranova and I seem to have a problem with the latest version and fails with the following error:
module github.com/terraform-aws-modules/terraform-aws-eks@latest found (v13.0.0+incompatible), but does not contain package github.com/terraform-aws-modules/terraform-aws-eks

Is there a specific version **terraform-aws-eks** that works with terranova?

**Comment by johandry** [comment]
Hi @hashir91,

I think `terraform-aws-modules/terraform-aws-eks` is not a provider, it's a module. The modules is a feature that is not supported on Terranova yet.

I'm working on the example 07 about Terraform modules but it requires some new changes in Terranova. I'm linking this issue with that branch/pull request example, so you will be notified when it's done.

**Comment by hashir91** [comment]
Okay, thanks for the reply. So, is there currently any way to use aws eks from Golang until we have the above mentioned proposed change?

**Comment by johandry** [comment]
Yes, there is. You can create an EKS cluster using the resources `aws_eks_node_group` and `aws_eks_cluster`, you need to create a few `iam` resources as well. There are some examples and documentation about this online. 

I did this long time ago and it worked. I'm working on an example using this old code so you can use it as a guide. I would like to finish the example by Friday, but as this is critical for you my suggestion is to look for examples using the EKS and `iam` resources and use them in your code.

Other option is to get into the code of the module `terraform-aws-modules/terraform-aws-eks`, take out what you need and put it in your code. I have not tried this option.

**Comment by russellsimpkins** [comment]
Have you had any time to get modules working? I thought your project was brilliant and wanted to use it, but ran into a roadblock on modules. I'm trying to create an API that uses terraform to create VPCs and EKS clusters. I've got it working in go using os/exec but I think it would be cleaner to do this with terranova. Unfortunately, I'm getting the "you didn't initialize module foo" error.
